commands/src/main/java/org/wildfly/extras/creaper/commands/auditlog/ChangeAuditLogger.java

```diff
@@ -91,30 +102,62 @@ public Builder logReadOnly(boolean logReadOnly) {
             return this;
         }
 
-        public Builder handler(String handler) {
+        public Builder addHandler(String handler) {
             if (handler == null) {
                 throw new IllegalArgumentException("handler can not be null");
             }
-            if (this.handlers == null) {
-                this.handlers = new ArrayList<String>();
+            if (this.addHandlers == null) {
+                this.addHandlers = new ArrayList<String>();
             }
-            this.handlers.add(handler);
+            this.addHandlers.add(handler);
             return this;
         }
 
-        public Builder handlers(String... handlers) {
+        public Builder addHandlers(String... handlers) {
             if (handlers == null) {
                 throw new IllegalArgumentException("handlers can not be null");
             }
-            if (this.handlers == null) {
-                this.handlers = new ArrayList<String>();
+            if (this.addHandlers == null) {
+                this.addHandlers = new ArrayList<String>();
+            }
+            this.addHandlers.addAll(Arrays.asList(handlers));
+            return this;
+        }
+
+        public Builder removeHandler(String handler) {
+            if (handler == null) {
+                throw new IllegalArgumentException("handler can not be null");
+            }
+            if (this.removeHandlers == null) {
+                this.removeHandlers = new ArrayList<String>();
             }
-            this.handlers.addAll(Arrays.asList(handlers));
+            this.removeHandlers.add(handler);
+            return this;
+        }
+
+        public Builder removeHandlers(String... handlers) {
+            if (handlers == null) {
+                throw new IllegalArgumentException("handlers can not be null");
+            }
+            if (this.removeHandlers == null) {
+                this.removeHandlers = new ArrayList<String>();
+            }
+            this.removeHandlers.addAll(Arrays.asList(handlers));
             return this;
         }
 
         public ChangeAuditLogger build() {
+            if (addHandlers != null && removeHandlers != null) {
+                List<String> auxList = new ArrayList<String>();
+                auxList.addAll(addHandlers);
+                auxList.removeAll(removeHandlers);
+                if (auxList.size() < addHandlers.size()) {
```

Comment:
There's also `Sets.intersection` in Guava, but that requires to create _two_ sets instead of one, so I'd prefer to use what I wrote above.